assessment.js

```diff
@@ -1,8 +1,12 @@
+'use strict'
 //We'll learn about require later in the course
-var ask = require('./ask.js');
+let ask = require('./ask.js');
 
-var answer = 'not empty';
 
-while (answer !== '' && answer !== 'SeCrEt') {
-  answer = ask("Guess my secret? ");
+// while (answer !== '' && answer !== 'SeCrEt') {
+//   answer = ask("Guess my secret? ");
+// }
+let answer = 0;
+while (answer !== "" && answer < 10 && answer !==2){
```

Comment:
Yes this would work but you wouldn't get confirmation that you got it right. Creative approach though.